### ISSUE TITLE
ERR_EMPTY_RESPONSE while loading local assets

### ISSUE THREAD

**Issue by ClementParis016** [balderdashy/sails#2958]
Hi there, I'm trying to run a Sails chat application on a Vagrant machine (Ubuntu guest on Windows host) and I get a very strange problem regarding assets loading.

The server can't load the local assets (js, css), it simply loads the page during a quite long time (+10 sec) and then returns an ERR_EMPTY_RESPONSE for each one. Only local files (I mean the assets of the Sails app) are affected, not the external ressources (i.e. external HTTP requests, like Google Fonts). Of course, I can't neither load an asset directly by typing its path in the browser, the error is the same.

My files are properly compiled to `.tmp/public/` folder. I don't think the problem comes from the Vagrant environment, because I can't even get these assets from the guest machine browser. Also I've tested with multiple browsers and the result is the same, but my colleague can run the app without any problem on his Macbook.

Does anybody could give me some help with that? Thanks.

Versions:
Sails 0.11.0
Node 0.10.32
Npm 1.4.28

**Comment by sailsbot** [comment]
Thanks for posting, @ClementParis016.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
